projects/plugins/jetpack/extensions/blocks/google-calendar/test/validate.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import { name, settings } from '../';
+import runBlockFixtureTests from '../../../shared/test/block-fixtures';
+
+/**
+ * 👀 Update this array of blocks to contain the name and settings for **ALL**
```

Comment:
It doesn't hurt to leave these comments here, but shall we remove them to keep it tidy?

Reply:
That was an oversight on my part. I'll clean it up.

Reply:
Once we finish adding fixtures and tests for other blocks, giving the code base another sweep to remove these might pay dividends. I see we have a few more blocks where these comments were left in place.